.drone.yml

```diff
@@ -4,15 +4,13 @@
 # ---
 kind: pipeline
 type: kubernetes
-name: test-chat-server
+name: test-all
 
 trigger:
```

Comment:
note: removing this allows all pushes to branches on mongodb/docs-chatbot to trigger test runs